flang/test/Evaluate/rewrite-out_of_range.F90

```diff
@@ -1,5 +1,7 @@
 ! Tests rewriting of OUT_OF_RANGE()
-! RUN: %flang_fc1 -fdebug-unparse -cpp %s 2>&1 | FileCheck %s --check-prefixes=CHECK%if target=x86_64{{.*}} %{,CHECK-X86-64%}
+! REQUIRES: target=x86-64{{.*}}
```

Comment:
@klausler did you mean `x86_64` (with an underscore instead of a hyphen) like you did in the predicate of line 4?

(The test is not being run at all as `unsupported`, not even on `x86_64`)

Reply:
Good catch!
https://github.com/llvm/llvm-project/pull/126699